package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "event-maker-telegram-bot",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
```

Comment:
what about the dev command

Reply:
Not sure what is the best option in rollup's case, so I'll let you to implement this, as you wanted to get familiar with rollup.

This issue might be helpful: https://github.com/rollup/rollup/issues/2385